web-admin/src/features/authentication/AvatarButton.svelte

```diff
@@ -32,64 +33,45 @@
   }
 
   const isDev = process.env.NODE_ENV === "development";
-
-  // Position the Menu popover
-  const [popperRef1, popperContent1] = createPopperActions();
-  const popperOptions1 = {
-    placement: "bottom-end",
-    strategy: "fixed",
-    modifiers: [{ name: "offset", options: { offset: [0, 4] } }],
-  };
-
-  // Position the View As User popover
-  const [popperRef2, popperContent2] = createPopperActions();
 </script>
 
-<Popover class="relative" let:close={close1}>
-  <PopoverButton use={[popperRef1]}>
+<DropdownMenu>
+  <DropdownMenuTrigger>
     <img
       src={$user.data?.user?.photoUrl}
       alt="avatar"
       class="h-7 inline-flex items-center rounded-full cursor-pointer"
       referrerpolicy={isDev ? "no-referrer" : ""}
     />
-  </PopoverButton>
-  <PopoverPanel
-    use={[popperRef2, [popperContent1, popperOptions1]]}
-    class="max-w-fit absolute z-[1000]"
-  >
-    <Menu minWidth="0px" focusOnMount={false} paddingBottom={0} paddingTop={0}>
-      {#if $page.params.organization && $page.params.project && $page.params.dashboard}
-        <ProjectAccessControls
-          organization={$page.params.organization}
-          project={$page.params.project}
-        >
-          <svelte:fragment slot="manage-project">
-            <ViewAsUserMenuItem
-              popperContent={popperContent2}
-              on:select-user={() => close1(undefined)}
-            />
-          </svelte:fragment>
-        </ProjectAccessControls>
-      {/if}
-
-      <MenuItem
-        focusOnMount={false}
-        on:select={() => {
-          // handleClose();
-          handleDocumentation();
-        }}>Documentation</MenuItem
-      >
-      <MenuItem focusOnMount={false} on:select={() => handleAskForHelp()}
-        >Ask for help</MenuItem
-      >
-      <MenuItem
-        focusOnMount={false}
-        on:select={() => {
-          // handleClose();
-          handleLogOut();
-        }}>Logout</MenuItem
+  </DropdownMenuTrigger>
+  <DropdownMenuContent>
+    {#if $page.params.organization && $page.params.project && $page.params.dashboard}
+      <ProjectAccessControls
+        organization={$page.params.organization}
+        project={$page.params.project}
       >
-    </Menu>
-  </PopoverPanel>
-</Popover>
+        <svelte:fragment slot="manage-project">
+          <!-- TODO: open this on click, rather than on hover  -->
+          <DropdownMenuSub>
+            <DropdownMenuSubTrigger>View as</DropdownMenuSubTrigger>
+            <DropdownMenuSubContent
+              class="min-w-[150px] max-w-[300px] min-h-[150px] max-h-[190px]"
```

Comment:
The ViewAsUserPopover will overflow without changing the layout algorithm for this. You can just add `flex flex-col` to the class.

<img width="441" alt="Screenshot 2023-12-21 at 2 07 01 PM" src="https://github.com/rilldata/rill/assets/120223836/ff388bd3-7918-42d3-a189-bd8188201397">

Reply:
Ah, good catch. Fixed.